Source/DataSources/EntityCluster.js

```diff
@@ -963,7 +963,8 @@ EntityCluster.prototype.destroy = function () {
  * @callback EntityCluster.newClusterCallback
  *
  * @param {Entity[]} clusteredEntities An array of the entities contained in the cluster.
- * @param {Object} cluster An object containing billboard, label, and point properties. The values are the same as
+ * @param {{billboard: Billboard, label: Label, point: PointPrimitive}} cluster An object
```

Comment:
I get the intent here but this just ends up being an `any` in the type definition either way.

Reply:
I totally didn't notice that.  Was I right about the actual shape of the object? I can fix it to use the  correct `@param` syntax, I'm just spoiled by the shorthand TS lets you use.

Reply:
Yes, the shape is correct.